test/IntegrationTests/MSTest.Acceptance.IntegrationTests/ThreadContextTests.cs

```diff
@@ -15,7 +15,7 @@ public async Task ThreadingContext_WhenCultureIsNotSet_TestMethodFails(string tf
     {
         var testHost = TestHost.LocateFrom(AssetFixture.InitToTestProjectPath, TestAssetFixture.InitToTestProjectName, tfm);
         TestHostResult testHostResult = await testHost.ExecuteAsync();
-        testHostResult.AssertOutputContainsSummary(failed: 1, passed: 0, skipped: 0);
+        testHostResult.AssertOutputContainsSummary(failed: 2, passed: 0, skipped: 0);
```

Comment:
Test methods are duplicated because the first method has the correct context automatically, but the second method has to restore it. So the first one would not fail, but second one would.

Reply:
Hence he higher test count.